cmake/templates/pkgConfig.cmake.in

```diff
@@ -95,8 +95,17 @@ if(NOT "@PROJECT_CMAKE_CONFIG_INCLUDE_DIRS@ " STREQUAL " ")
   set(@PROJECT_NAME@_INCLUDE_DIRS "")
   set(_include_dirs "@PROJECT_CMAKE_CONFIG_INCLUDE_DIRS@")
   foreach(idir ${_include_dirs})
-    if(IS_ABSOLUTE ${idir} AND IS_DIRECTORY ${idir})
-      set(include ${idir})
+    if(IS_ABSOLUTE ${idir})
+      get_filename_component(path ${idir} DIRECTORY)
+      get_filename_component(idir ${idir} NAME)
+      set(include "include-NOTFOUND")
+      find_path(include ${idir}
+        PATHS ${path}
+        NO_DEFAULT_PATH)
+      if(NOT include)
+        message(FATAL_ERROR "Project '@PROJECT_NAME@' specifies '${idir}' as an include dir, which is not found.  It does neither exist as an absolute directory nor in '@PKG_INCLUDE_PREFIX@/${idir}'.  Ask the maintainer '@PROJECT_MAINTAINER@' to fix it.")
```

Comment:
In case my previous assumption about `find_path` is right this error message seems weird. We had an absolute path before, why would the message only mention the folder name? Also even if the name is not found in any root path it should always be available under the original absolute path. So why should this happen anyway?

If I understand it correctly you are using `find_path` here to find the same absolute path nested under a potential root path, right?